src/app/profile/update/update.component.html

```diff
@@ -82,6 +82,11 @@ <h1>Edit Profile</h1>
             <span class="fields-status-pf">characters remaining</span>
           </span>
         </div>
+        <div class="form-group">
+          <button class="btn btn-default" id="passwordReset">
+            Change password <i class="fa fa-external-link"></i>
```

Comment:
This is the link we should use here - https://developers.redhat.com/auth/realms/rhd/account/password
Could you update your PR with that link?

Reply:
Added - tested locally and it brought me to the proper password reset page.